blaze/compute/sql.py

```diff
@@ -871,7 +871,12 @@ def compute_up(expr, data, **kwargs):
 @dispatch(DateTime, (ClauseElement, sa.sql.elements.ColumnElement))
 def compute_up(expr, data, **kwargs):
     if expr.attr == 'date':
-        return sa.func.date(data).label(expr._name)
+        col, = get_inner_columns(data)
+        d = sa.func.date(col).label(expr._name)
+        # Manually reassign the bind, cannot pass to function:
+        # https://bitbucket.org/zzzeek/sqlalchemy/issues/3519
+        d.bind = data.bind
```

Comment:
is this going to be a problem with computing on any function, i.e., not just calling `date` on a function?